### PR TITLE
chore: updated lng to lon

### LINKED ISSUE #127: Update `lng` to `lon` for consistancy
# Description
Currently we are using both `lon` and `lng` in our code.
For consistancy we should update `lng` to `lon` since it is the most commonly used.

Reason: in the Web Map all the entities (trees) are using `lon` for coordinates.

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-core#treetracker-web-map-core) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.

### PR DESCRIPTION
**Pull request by muhon9**
Description

Updated all lng to lon for consistency 

Issue(s) addressed

Resolves #127

### CODE REVIEW

**Review by RubenSmn — APPROVED** [review]
Great work!

File: src/Map.js [hunk 1]
```diff
@@ -987,7 +987,7 @@ export default class Map {
     log.log('current center:', center)
     const zoom_level = this.map.getZoom()
     const res = await this.requester.request({
-      url: `${this.apiServerUrl}nearest?zoom_level=${zoom_level}&lat=${center.lat}&lng=${center.lng}`,
+      url: `${this.apiServerUrl}nearest?zoom_level=${zoom_level}&lat=${center.lat}&lon=${center.lon}`,
```
Comment (dadiorchen) [inline src/Map.js:990]: @RubenSmn seems this requirs the change on API side too?

**Review by dadiorchen — COMMENTED** [review]
@RubenSmn please test this in the sandbox, there might be problem if the server side is returning lon rather then lng

File: src/Map.js (hunk 1 above)
Comment (RubenSmn) [reply src/Map.js:990]: The problem is also that the `center` does return a `lng`

Comment (dadiorchen) [reply src/Map.js:990]: Okay, we can do this way: revert this PR and re-create it, then let's go to the API side and finish the correction, then when can merge this PR, overwise, this PR will cause problem, right?

Comment (RubenSmn) [reply src/Map.js:990]: Could we update the API so that this will be fixed?

Comment (RubenSmn) [reply src/Map.js:990]: I created a PR for the API https://github.com/Greenstand/treetracker-web-map-api/pull/353

Comment (dadiorchen) [reply src/Map.js:990]: Great, I'll take a look, but I vaguely remember it's the PostgreSQL return `lon` so we align with it's convention maybe, you can try this:

```
select region_id id, ST_ASGeoJson(centroid) centroid from active_tree_region, region 
where region.id = active_tree_region.region_id
```

To check the return json from postgresql

Comment (dadiorchen) [reply src/Map.js:990]: Maybe there are some postgresql function can do some traslation

Comment (RubenSmn) [reply src/Map.js:990]: @dadiorchen just checked and the test is passing!

Comment (muhon9) [reply src/Map.js:990]: @dadiorchen @RubenSmn is there anything i have to do?

**Pull request merged**